README.md

```diff
@@ -20,6 +20,17 @@ Simply add something similar to this to your styles:
   height: 400px;
 }
 ```
+In case you want to make the google map to have full width and height as the parent div, you can do the following:
+
+```css
+.g-map{
+  height:100%;
+}
+.g-map-canvas {
+  width:100%;
```

Comment:
sorry, but could you add space after `:` here and 3 lines above? 
and then I'll merge 👍